src/components/about-us/Timeline.tsx

```diff
@@ -1,32 +1,65 @@
-import { FC } from "react";
-import "@/stylesheets/timeline.css";
+import React from "react";
+import Images from "../image/Image";
+import classNames from "classnames";
 
-type TimelineItem = {
+interface TimelineItemProps {
+  imageSrc: string; // Image source
   period: string;
-  description: string;
-};
+  bgColor: string; // Background color of the parent container
+  description: string; // Description text
+}
 
-type Props = {
-  items: TimelineItem[];
-};
+interface TimelineProps {
+  items: TimelineItemProps[];
+}
 
-const Timeline: FC<Props> = ({ items }) => (
-  <section className="px-4 pt-20 max-w-6xl mx-auto">
-    <h2 className="text-4xl font-semibold mb-4">Our history</h2>
-    <div className="timeline-container">
-      {items.map((item, i) => (
-        <div
-          key={`item-${i}`}
-          className={`timeline-item ${i % 2 === 0 ? "left" : "right"}`}
-        >
-          <p className="text-xl text-navy-600 font-bold mb-2 timeline-title">
-            {item.period}
-          </p>
-          <p>{item.description}</p>
-        </div>
-      ))}
-    </div>
-  </section>
-);
+const TimelineItem: React.FC<TimelineProps> = ({ items }) => {
+  return (
+    <>
+      <div className="text-[48px] font-bold leading-[56.25px] tracking-[-0.16px] mt-[60px] mb-[40px] text-center text-navy-900">
+        Our history
+      </div>
+      <div
+        className="md:space-y-[-60px] space-y-3 mb-24 overflow-hidden md:mx-32 
+                relative"
+      >
+        {items.map((item, index) => (
+          <div
+            className="relative flex items-center inset-0 justify-between md:justify-normal md:even:flex-row-reverse group mx-2 sm:mx-10 md:mx-0"
+            key={index}
+          >
+            {items.length - 1 !== index && (
+              <div className="absolute top-[50px] translate-x-[49px] md:-translate-x-1/2 md:left-1/2 h-full w-[3px] bg-navy-900"></div>
+            )}
+            <div
+              className={classNames(
+                "text-[15px] lg:text-[20px] text-center font-semibold leading-[23.44px] absolute left-3 top-3 translate-x-0 md:left-1/2 md:-translate-x-1/2 md:top-[50px] w-[80px] h-[80px] lg:w-[100px] lg:h-[100px] rounded-full border border-white shadow-md flex items-center justify-center flex-col",
+                item.bgColor === "#4362A6" ? "text-gray-50" : "turquoise-600",
```

Comment:
**question (blocking):** Would it just be easier to specify the Tailwind color name for `item.bgColor` rather than a hex code which you have to select the color name from in the code?